server/lib/realtime/resource/http.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Realtime.Resource.Http do
+  @moduledoc """
+  This resource handler executes HTTP calls and returns their response.
+
+  ## Arguments
+
+   * `url` - The request URL
+   * `method` - The request HTTP method
+   * `headers` - A map of key-values that will be added to the request headers
+   * `body` - The request body
+
+  ## Result
+
+   * `headers` - The response headers
+   * `status_code` - The response status code
+   * `body` - The response body
```

Comment:
I know this is very very rough implementation yet but I wanted to point out some things that might be useful 

 - response body can contain non-serializable data. 
 - there might not be any data at all
 - data can be so big that leading to memory overflow 
 
 these cases could prevent the response body to be passed as input to the next state.

Reply:
thanks! will definitely keep these things in mind as we continue to develop this resource.

Reply:
I am sorry I know that this PR is not a discussion place but I really want to get involved in the development.

I really wonder, how these flows will be executed. (as I can see there I there is a new thread for every new execution) and what will happen when there is an enormous back pressuring queue of tasks? 

I don’t think having many threads will solve the problem because after the limits of the host machine there will be no space to execute any more workflows.

there will be a need for horizontal scaling after some time imho.

Reply:
> really want to get involved in the development

appreciate that you want to get involved!

> I really wonder, how these flows will be executed.

I think for the initial version we just want to make sure it works with a couple of resources like webhooks and email so all workflows will be transient. I think you're exactly right when you identified a new process for every execution and an enormous queue of tasks on the host machine causing problems down the line. We've discussed using Postgres-backed queues (Oban) and a pool of processes for workflow executions but we'll have to investigate how well that scales and explore viable alternatives.